generate-metadata/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "generate-metadata",
+  "version": "1.0.0",
+  "description": "",
+  "main": "GenerateMetadata.js",
+  "type": "module",
+  "scripts": {
+    "build": "rm -f /app/grassroots-backend/src/grassroots-shared/metadata.ts && swc src --strip-leading-paths --out-dir dist && node dist/GenerateMetadata.js"
```

Comment:
Calling this build is a bit weird. Again, we could fix this, but it probably makes more sense to fix it once we're using turborepo. 

I didn't want to introduce turborepo in this patch, as it's already huge.